looprpc/server.proto

```diff
@@ -51,6 +56,9 @@ message ServerLoopOutQuote {
     uint64 max_swap_amount = 6;
 
     int32 cltv_delta = 7;
+
+    /// The unix time in seconds we want the swap to be initiated by.
+    int64 max_swap_wait_time_unix = 8;
```

Comment:
This is part of the request?

Maybe we should also now add `swap_amt` to the request and return `swap_fee` in the response. We can then deprecate `swap_fee_base` and `swap_fee_rate`, as a preparation for more advanced pricing models.

Reply:
can be done in later PR, but yeah that sounds reasonable.

Reply:
Ok, but `max_swap_wait_time_unix` should still be part of the quote request? Otherwise, what is returned here by the server?

Reply:
`swap_publication_deadline` is in the quote request, if that is what you mean?

Reply:
it is currently in `ServerLoopOutQuote`, but I thought it should be added to `ServerLoopOutQuoteRequest`?

Reply:
oh, yeah that's a mistake 🤦‍♂

Reply:
Can even just consider removing it for now? As it is not used anywhere currently